src/components/Card/ProductCard/ProductCard.jsx

```diff
@@ -1,27 +1,60 @@
 import React from 'react'
+import { BiXCircle } from "../../../assets/icons";
 
-function ProductCard({badge,img,description,isBtnSaveToWishlist=false,originalPrice,name,discountedPrice,discountPercent,icon,...rest}) {
+function ProductCard({
+  badge,
+  img,
+  isDismissBtn = false,
+  description,
+  isBtnSaveToWishlist = false,
+  originalPrice,
+  name,
+  discountedPrice,
+  discountPercent,
+  icon,
+  ...rest
```

Comment:
not sure if it is a good practice to use spread operator here.

Reply:
It is a rest parameter not spread Balaji. And I am using it because in case I am passing something other than the mentioned props I can access it from the rest.

Reply:
yes my bad it is the rest parameter. okay what I understood is that you have de-structured the object in the map function and passed those values as props to your card component. I think it would be better to de-structure them inside the card component itself so that you need not handle so many props. Correct me if I have understood it in a wrong way.

Reply:
Yeah, thanks for suggesting that will implement it. Makes sense.